### PR TITLE
Copy GEANT4 loader

### LINKED ISSUE #11: Copy DREAM GEANT4 loader into essdiffraction

### PR DESCRIPTION
**Pull request by jl-wynen**
Fixes #11

### CODE REVIEW

File: requirements/nightly.in [hunk 1]
```diff
@@ -4,7 +4,7 @@
 dask
 graphviz
 pythreejs
-https://github.com/scipp/scipp/releases/download/nightly/scipp-nightly-cp39-cp39-manylinux_2_17_x86_64.manylinux2014_x86_64.whl
+https://github.com/scipp/scipp/releases/download/nightly/scipp-nightly-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl
```
Comment (SimonHeybrock) [inline requirements/nightly.in:7]: How come this is changed?

Comment (jl-wynen) [reply requirements/nightly.in:7]: Good question. All I did is run `tox -e deps`

Comment (SimonHeybrock) [reply requirements/nightly.in:7]: ... in the wrong env?

Comment (jl-wynen) [reply requirements/nightly.in]: I used python 3.10 to lock dependencies. I re did it with 3.9 now which reverted the change here.

**Review by SimonHeybrock — APPROVED** [review]

**Pull request merged**